src/Microsoft.Health.Fhir.Api/Features/Throttling/ThrottlingMiddleware.cs

```diff
@@ -78,14 +134,19 @@ public async Task Invoke(HttpContext context)
                 if (Interlocked.Increment(ref _requestsInFlight) <= _configuration.ConcurrentRequestLimit)
                 {
                     // Still within the concurrent request limit, let the request through.
+                    Interlocked.Increment(ref _currentPeriodSuccessCount);
```

Comment:
When do these values reset? It looks like they would be remembered as long as the server is running, so this count would be less useful the longer the server is up.

Reply:
I think with the above it resets to 0 when the SamplingLoop runs

Reply:
Yes, these are reset with every sample loop iteration
```csharp
var successCount = Interlocked.Exchange(ref _currentPeriodSuccessCount, 0);
var failureCount = Interlocked.Exchange(ref _currentPeriodRejectedCount, 0);
```